attributes/default.rb

```diff
@@ -7,18 +7,27 @@
   if ENV['ProgramFiles'] == ENV['ProgramFiles(x86)']
     # if user has never logged into an interactive session then ENV['homedrive'] will be nil
     default['zabbix']['etc_dir']    = ::File.join((ENV['homedrive'] || 'C:'), 'Program Files', 'Zabbix Agent')
+    default['zabbix']['install_dir']         = 'C:/opt/zabbix'
+    default['zabbix']['src_dir']             = 'C:/opt'
+    default['zabbix']['tmp_dir']             = 'C:/tmp'
   else
     default['zabbix']['etc_dir']    = ::File.join(ENV['ProgramFiles'], 'Zabbix Agent')
+    default['zabbix']['install_dir']         = 'C:/opt/zabbix'
```

Comment:
I'd recommend to install it to `C:\Zabbix` rather than `/opt`; avoids confusion when reading the code. `/opt` is usually linux world